src/main/scala/com/fulcrumgenomics/umi/FilterConsensusReads.scala

```diff
@@ -96,8 +96,8 @@ class FilterConsensusReads
   val maxReadErrorRate: Seq[Double] = Seq(0.025),
   @arg(flag="e", minElements=1, maxElements=3, doc="The maximum error rate for a single consensus base.")
   val maxBaseErrorRate: Seq[Double] = Seq(0.1),
-  @arg(flag="N", minElements=1, maxElements=3, doc="Mask (make 'N') consensus bases with quality less than this threshold.")
-  val minBaseQuality: PhredScore = DefaultMinConsensusBaseQuality,
+  @arg(flag="N", doc="Mask (make 'N') consensus bases with quality less than this threshold.")
+  val minBaseQuality: PhredScore,
```

Comment:
Why not set it to some reasonable low value (ex. `2.toByte`) so that the user can run the tool end-to-end.  There are enough filter options that have defaults.